conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+# FIX Connect
+
+This "Connect" component extension supports connection to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).
+
+## Available service types
+
+This "Connect" component provides users with two types of the services:
+
+### FIX Client
+
+The "Connect" component acts like a client that connects to the system.
+The type to use in the configuration - **FIX_Client**.
+
+#### Configurable parameters
+
+The FIX Client has following parameters to configure:
+```yaml
+# It is required param. Version of FIX this session should use.
+# Possible values: "FIX.4.0", "FIX.4.1", "FIX.4.2", "FIX.4.3", "FIX.4.4", "FIXT.1.1"
+beginString: FIXT.1.1
+
+# It is required param. Your ID as associated with this FIX session.
+senderCompID: sender
+
+# It is required param. Counter parties ID as associated with this FIX session.
+targetCompID: target
+
+# It is required param. Directory to store sequence number and message files inside the workspace directory.
+fileStorePath: store/fix/sessions
+
+# It is required param. Time of day that this FIX session becomes activated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between StartTime and EndTime the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. That means next day after the start at 'EndTime' the bounds will be moved
+# Format of value is 99:99:99
+startTime: 00:00:00
+
+# It is required param. Time of day that this FIX session becomes deactivated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between StartTime and EndTime the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. That means next day after the start at 'EndTime' the bounds will be moved
+# Format of value is 99:99:99
+endTime: 00:00:00
+
+# It is required param.
+# Specifies the default application version ID for the session.
+# This can either be the ApplVerID enum (see the ApplVerID field) or the BeginString for the default version.
+# Possible values: "2", "3", "4", "5", "6", "7", "8", "9"
+defaultApplVerID: 9
+
+# Determines if sequence numbers should be reset to 1 after a normal logout termination
```

Comment:
Determines if **the** sequence numbers should be reset to 1 **,** after a normal logout termination